updated/fixed de_DE.lang

Scoaex opened the pull request:
fixed typos, improved some translations that made no sense (or 2 different translations for 1 thing in achievements/blocks),
removed the "-" after ME (not needed, there is no in other lang files aswell),...

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -2,20 +2,20 @@
 tile.appliedenergistics2.crank.name=Holzkurbel
 tile.appliedenergistics2.creative_energy_cell.name=Kreative Energiezelle
-tile.appliedenergistics2.dense_energy_cell.name=Dichte Energiezelle
+tile.appliedenergistics2.dense_energy_cell.name=Komprimierte Energiezelle
```
Wrong. This is about high density and not compression.

Scoaex replied:
True, but the 1:1 translation makes no sense here in my opinion cause "Dicht" doesnt  really fit for that, thats why i looked for something better

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -25,12 +25,12 @@ tile.appliedenergistics2.quartz_glass.name=Quarzglas
-tile.appliedenergistics2.light_detector.name=Quarzlichtsensor
-tile.appliedenergistics2.spatial_io_port.name=IO-Raumport
+tile.appliedenergistics2.light_detector.name=Quarz Lichtsensor
+tile.appliedenergistics2.spatial_io_port.name=IO Port
```
It is still an spatial port, not a normal IO Port

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -25,12 +25,12 @@ tile.appliedenergistics2.quartz_glass.name=Quarzglas
+tile.appliedenergistics2.tiny_tnt.name=Mini TNT
 tile.appliedenergistics2.vibration_chamber.name=Vibrationskammer
-tile.appliedenergistics2.wireless_access_point.name=Drahtloser ME-Zugangspunkt
+tile.appliedenergistics2.wireless_access_point.name=ME Drahtloser Zugangspunkt
```
The order is somewhat wrong as this is a access point with the additional property of being wireless. Could also be a tethered one.

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -43,13 +43,13 @@ tile.appliedenergistics2.sky_stone_chest.name=Himmelssteintruhe
-tile.appliedenergistics2.crafting_accelerator.name=Fertigungs-Prozessoreinheit
-tile.appliedenergistics2.crafting_unit.name=Fertigungseinheit
-tile.appliedenergistics2.molecular_assembler.name=Molekularassembler
+tile.appliedenergistics2.crafting_storage_1k.name=1k Speicherzelle
```
Plain wrong. Still crafting storage not normal ones.

Scoaex replied:
Messed it up here with the cells, my bad

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -43,13 +43,13 @@ tile.appliedenergistics2.sky_stone_chest.name=Himmelssteintruhe
-tile.appliedenergistics2.crafting_unit.name=Fertigungseinheit
-tile.appliedenergistics2.molecular_assembler.name=Molekularassembler
+tile.appliedenergistics2.crafting_storage_1k.name=1k Speicherzelle
+tile.appliedenergistics2.crafting_storage_4k.name=4k Speicherzelle
```
Same

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -43,13 +43,13 @@ tile.appliedenergistics2.sky_stone_chest.name=Himmelssteintruhe
-tile.appliedenergistics2.molecular_assembler.name=Molekularassembler
+tile.appliedenergistics2.crafting_storage_1k.name=1k Speicherzelle
+tile.appliedenergistics2.crafting_storage_4k.name=4k Speicherzelle
+tile.appliedenergistics2.crafting_storage_16k.name=16k Speicherzelle
```
Same

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -43,13 +43,13 @@ tile.appliedenergistics2.sky_stone_chest.name=Himmelssteintruhe
+tile.appliedenergistics2.crafting_storage_1k.name=1k Speicherzelle
+tile.appliedenergistics2.crafting_storage_4k.name=4k Speicherzelle
+tile.appliedenergistics2.crafting_storage_16k.name=16k Speicherzelle
+tile.appliedenergistics2.crafting_storage_64k.name=64k Speicherzelle
```
Same

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -293,33 +295,33 @@ gui.tooltips.appliedenergistics2.MoveWhenWorkIsDone=Auswerfen wenn fertig.
 gui.tooltips.appliedenergistics2.Disabled=Deaktiviert
 gui.tooltips.appliedenergistics2.Enable=Aktiviert
-gui.tooltips.appliedenergistics2.Blocking=Nicht fertigen, wenn Zielinventar Items enthält.
+gui.tooltips.appliedenergistics2.Blocking=Keine Fertigungsitems einfügen während das Zielinventar Items enthält.
```
"Fertigungsitems" is not really a word. At least it should be "Gegenstand" or whatever the official translation for item is used by minecraft.

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -417,45 +419,45 @@ item.appliedenergistics2.material.Singularity.name=Singularität
-item.appliedenergistics2.multi_part.ConversionMonitor.name=ME-Konversionsmonitor
-item.appliedenergistics2.multi_part.CraftingTerminal.name=ME-Fertigungskonsole
+item.appliedenergistics2.multi_part.CableSmart.name=Schlaues ME Kabel
+item.appliedenergistics2.multi_part.CableDense.name=Dickes ME Kabel
```
Also about density not size.

Scoaex replied:
Same with Energy Cell, used "Dick" here cause it made more sense here
I´ll change it back to "Dicht" until someone comes up with a better translation

M4lik replied:
I would also replace "Schlaues ME Kabel" with "Itelligentes ME Kabel"...

yueh commented on src/main/resources/assets/appliedenergistics2/lang/de_DE.lang:
```diff
@@ -487,10 +489,10 @@ item.appliedenergistics2.matter_cannon.name=Materiekanone
 
-item.appliedenergistics2.debug_card.name=Dev.Debug-Karte
-item.appliedenergistics2.debug_replicator_card.name=Dev.Replikator-Karte
+item.appiedenergistics2.debug_card.name=Dev.Debug Karte
```
`appliedenergistics2`

Scoaex commented:
Fixed my typos and that i accidently mixed up crafting storage and the storage cells.
Also reverted to the old "Dense" translation as you said, maybe someone comes up with something better since the 1:1 translation doesnt really fit here in my opinion.

bolli24 commented:
I would use "kompakt" instead of "dicht". In my opinion this fits the best.
